roles/repository/templates/alfresco-global.properties.j2

```diff
@@ -68,6 +68,25 @@ dsync.service.uris={% if use_ssl | bool %}https{% else %}http{% endif %}://{% if
 
 aos.baseUrlOverwrite={% if use_ssl | bool %}https{% else %}http{% endif %}://{% if fqdn_alfresco %}{{ fqdn_alfresco }}{% else %}{{ nginx_host }}{% endif %}:{% if use_ssl | bool %}{{ ports_cfg.nginx.https }}{% else %}{{ ports_cfg.nginx.http }}{% endif %}/alfresco/aos
 
+# CSRF filter overrides
+{% import 'csrf_macros.j2' as _csrf %}
+csrf.filter.enabled={% if csrf.enabled %}true
+{% if csrf.urls -%}
+  csrf.filter.referer={{ _csrf.compute('referer', csrf.urls)}}
+{% endif %}
+csrf.filter.origin={{ _csrf.compute('origin', csrf.urls)}}
+csrf.filter.referer.always={% if 'origin' in csrf.force_headers | lower %}true
+{% else -%}
+  false
+{% endif -%}
+csrf.filter.origin.always={% if 'referer' in csrf.force_headers | lower %}true
+{% else -%}
+  false
+{% endif -%}
+{% else -%}
+  false
+{% endif %}
```

Comment:
it's barely readable with all this nested conditionals

maybe it's better to construct a key-value structure and here in the template just foreach it like at L90

Reply:
Doing so we would only move that templating mess in an ansible task or variable definition as we would anyway  use jinja templating.
I'm not sure that's worth the effort tbh

Reply:
🙏🏻